src/views/dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from "react";
+import DarkMode from "../_shared/DarkMode";
+import Footer from "../_shared/Footer";
+import "../home/Home";
+import Logout from "../_shared/Logout";
+import SocialIcons from "../_shared/SocialIcons";
+import { Project } from "./ProjectForm";
+import User from "./User";
+//import Loding from "../_shared/Loading";
+
+const Dashboard = props => {
+   const [name, setName] = useState("Login please");
+
+   useEffect(() => {
+      const username = localStorage.getItem("username");
+
+      if (username) {
+         setName(username);
+      }
+   }, []);
+
+   return (
+      <>
+         <div className="App">
+            <main className="section-main">
+               <DarkMode />
```

Comment:
We could probably remove Dark Mode from the page here and just create a User Settings area where they can set preferences.

Reply:
Great idea! I like that